lms/routes.py

```diff
@@ -4,6 +4,7 @@
 
 def includeme(config):
     config.add_route("index", "/")
+    config.add_route("feature_flags_test", "/flags/test")
```

Comment:
I've added a test view so that I can test it on QA and prod. I'll send a separate PR to remove this later